src/ApiService/ApiService/Functions/AgentRegistration.cs

```diff
@@ -134,7 +134,10 @@ public class AgentRegistration {
             ScalesetId: scalesetId,
             Version: version);
 
-        await _context.NodeOperations.Replace(node);
+        var r = await _context.NodeOperations.Replace(node);
+        if (!r.IsOk) {
+            _log.WithHttpStatus(r.ErrorV).Error($"failed to replace node operations for node {node.MachineId}");
+        }
```

Comment:
does the machineId show up as a field in the customDimension?

Reply:
no it's part of the message

I will redo logging later on to process formatted strings and then it will show up in custom dimensions (in this specific case I'll do it manually)

Reply:
Yeah, it's just really handy to have those kinds of fields factored out.